### PR TITLE
Post: MkDocs YAML manipulation

### PR DESCRIPTION
**Pull request by humitos**
Blog post announcing the depreaction of the `mkdocs.yml` manipulation done by Read the Docs at build time, following the plan described.

Related:
* https://github.com/readthedocs/addons/issues/72#issuecomment-1926647293
* https://github.com/readthedocs/readthedocs.org/pull/11206
* https://github.com/readthedocs/readthedocs.org/issues/8529

----
📚 Documentation preview 📚: https://readthedocs-about--277.org.readthedocs.build/blog/2024/03/mkdocs-yaml-manipulation/

### CODE REVIEW

File: content/posts/mkdocs-yaml-manipulation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+title: Stop manipulating "mkdocs.yml"
```
Comment (ericholscher) [inline content/posts/mkdocs-yaml-manipulation.md:1]:
```suggestion
title: Build process change: removing changes to mkdocs.yml
```

File: content/posts/mkdocs-yaml-manipulation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+category: Changelog
+tags: builders, deprecation, mkdocs, addons
+authors: Manuel Kaufmann
+status: published
+image: /images/mkdocs-yaml-manipulation.jpg
+image_credit: Photo by <a href="https://unsplash.com/@flowforfrank?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Ferenc Almasi</a> on <a href="https://unsplash.com/photos/text-yXrl2lkGJ-k?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+We are announcing the deprecation of the `mkdocs.yml` manipulation done by Read the Docs at build time on **Monday, April 15th, 2024**.
+After this date, there will be some changes in the behavior:
```
Comment (ericholscher) [inline content/posts/mkdocs-yaml-manipulation.md:12]:
```suggestion
After this date, there will be some changes in build process behavior:
```

File: content/posts/mkdocs-yaml-manipulation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* MkDocs default theme will be used if your documentation is not defining one on its `mkdocs.yml` file.
+  If you want to continue using the `readthedocs` theme, read [Choosing your Theme](https://www.mkdocs.org/user-guide/choosing-your-theme/) MkDocs' documentation
+  to learn how to explicity define it in the configuration file.
+* [The new beta Read the Docs Addons](https://blog.readthedocs.com/addons-flyout-menu-beta/) will be enabled by default.
+
+
+We've made this decision towards a simplification of the build process executed on Read th Docs,
+allowing our users to get the same results when building their documentation locally and on our platform.
+Besides, this simplification allows more complex configuration in the YAML by using the special `!!` syntax
+[that wasn't supported due to security reasons](https://github.com/readthedocs/readthedocs.org/issues/8529).
```
Comment (ericholscher) [inline content/posts/mkdocs-yaml-manipulation.md:23]:
```suggestion
We've made this decision to simplify the build process executed on Read the Docs,
allowing our users to get the same results when building their documentation locally and on our platform.
This simplification allows more complex configuration in the YAML by using the special `!!` syntax
[that wasn't previously supported due to security reasons](https://github.com/readthedocs/readthedocs.org/issues/8529).
```

**Review by ericholscher — APPROVED** [review]
Looks good -- we should plan to email these users about this change, and probably update [the GH issue](https://github.com/readthedocs/readthedocs.org/issues/10587) tracking these ongoing deprecations.

**Comment by humitos** [comment]
> we should plan to email these users about this change

I created an email template based on this blog post at https://hackmd.io/LSQ28s4sTImKw35o08958g. If you give me the 👍🏼 I can send it in the next days.

> and probably update [the GH issue](https://github.com/readthedocs/readthedocs.org/issues/10587) tracking these ongoing deprecations.

Done ✅

**Comment by ericholscher** [comment]
@humitos I don't have permission on that hackmd.

**Comment by humitos** [comment]
@ericholscher ouch, I just sent you the invite 👍🏼

**Comment by ericholscher** [comment]
@humitos email looks good.

**Comment by humitos** [comment]
Sent 📧

**Pull request merged**